resources/settings.xml

```diff
@@ -125,5 +125,31 @@
                 </setting>
             </group>
         </category>
+        <category id="addons_options" label="30240" help="">
+            <group id="1" label="30240">
+                <setting id="upnext_mode" type="string" label="30090" help="">
+                    <level>0</level>
+                    <default>disabled</default>
+                    <constraints>
+                        <options>
+                            <option label="30091">best</option>
```

Comment:
I think these options need more explanation in the Settings GUI, as it is not self-explaining.

Reply:
In fact, it may be named like that (because, this is what it does):

"Show UpNext dialog at":
* "credits start"
* "preview start"
* "fixed time before the end"
* "never (disabled)"